### ISSUE TITLE
Initial Build Failure - "file integrity checksum failed"

### ISSUE THREAD

**Issue by mikerossms** [microsoft/AzureTRE#4041]
**Issue**
Following the quick deployment guide, running make all terminates with:

file integrity checksum failed for "opt/startup/install_ca_certificates.sh"
make: *** [/home/vscode/AzureTRE/Makefile:97: push-airlock-processor] Error 1

This error happens every time (been tested over consecutive days) and I have been unable to find a resolution at this stage.

**Steps to reproduce**

1. From a clean build run all steps including "make auth"
2. On success added admin to the App TRE API
3. Ran "make all"

**Azure TRE release version:** 0.18.0

**Build setup**
Azure Windows 11 VM with nested hypervisor (Standard_D4as_v5)
Docker Desktop 4.32
64GB Disk (with plenty of free space)

It is behind an azure firewall, but has full outbound access on 80 and 443.  Logs are clean and show no deny.

**Comment by tim-allen-ck** [comment]
Hi @mikerossms
It looks like Make failed on the push-airlock-processor step
Do think you could run `make build-and-push-airlock-processor` and let me know the output?

**Comment by mikerossms** [comment]
I am noticing a lot of "retrying" happening.  It starts the push and maybe gets about 1/3 to 1/2 of the way through then drops to retrying.

Checking for Docker...
»»» 🔨 Azure details from logged on user 
»»»   • Subscription: ---
»»»   • Tenant:       ---

Login Succeeded
The push refers to repository [acrtrecontainer.azurecr.io/microsoft/azuretre/airlock-processor]
252f1882e68f: Layer already exists 
4f51e812bde8: Layer already exists 
6d767f27b8e3: Layer already exists 
bb48cf2fb221: Pushing [==================================================>]  1.536kB
2df2a1f117fc: Retrying in 1 second 
68daf97674ee: Pushing [========================>                          ]  381.9MB/764.6MB
1caa1458f0a8: Layer already exists 
bf9f0b09f10e: Layer already exists 
f205fbd1e5ba: Retrying in 12 seconds 
11b0699f6a6d: Pushing [=====>                                             ]  34.72MB/336.3MB
c3e582b7a847: Waiting 
1d4f37514bff: Waiting 
aa171b297ed1: Waiting 
70208fd551ef: Waiting 
a2375faae132: Waiting 
file integrity checksum failed for "opt/startup/install_ca_certificates.sh"
make: *** [/home/vscode/AzureTRE/Makefile:97: push-airlock-processor] Error 1

**Comment by tim-allen-ck** [comment]
Could be that your docker client needs an increase in memory. 
In the app try going to Preferences->Advanced->Memory.
or try `docker system prune -a`

**Comment by mikerossms** [comment]
Adjusted memory from 2GB to 6GB (though .wslconfig and settings.json as only the Hyper-V version has the advanced resource settings for memory in Docker Desktop and I am running on WSLv2).  This did have an impact, however it did not solve the issue above.  I tried all of the following steps in this order:

1. Just running "" (peaked at 2.7GB)
2. Running "make all" (peaked at 4.92GB)
3. Clean build of VM from scratch with new Docker Desktop (set to 6GB), VSCode in dev container, make auth and make all

Only by running the last step was I able to get this up and running.  For some reason, the build ran fine :)

There is one note for this though, the build fails when you have deployed and Azure Landing Zone (ALZ) due to the default policies around Storage Accounts.  I had to modify bootstrap.sh and main.tf under devops/terraform and add "enable https traffic only" and "min TLS version - TLS1_2" otherwise ALZ policy stopped this from progressing.
